version/info.go

```diff
@@ -41,14 +43,16 @@ func NewCollector(program string) prometheus.Collector {
 			Namespace: program,
 			Name:      "build_info",
 			Help: fmt.Sprintf(
-				"A metric with a constant '1' value labeled by version, revision, branch, and goversion from which %s was built.",
+				"A metric with a constant '1' value labeled by version, revision, branch, goversion from which %s was built, and the goos and goarch for the build.",
 				program,
 			),
 			ConstLabels: prometheus.Labels{
 				"version":   Version,
 				"revision":  Revision,
 				"branch":    Branch,
 				"goversion": GoVersion,
+				"os":        GoOS,
```

Comment:
@roidelapluie Since this is going to affect a lot of components, just double-checking: are you good with just `os` and `arch` here vs. `goos` / `goarch`? Maybe we should use the latter in case the system is an `amd64` one but the binary is compiled for `386`, so it could be confusing to people if they think this label designates the arch of the underlying system?

Reply:
I think I'm a little bit confused then and why I wasn't sure about the help message 😅 
The binary can be compiled for a different arch? I assumed the arch would match the one of the system it's running on.

Reply:
So you can totally run a Go binary compiled for the `386` architecture on the (backwards-compatible but newer) `amd64` architecture. You can try it by doing:

```bash
GOARCH=386 go build main.go
./main.go
```

...on an `amd64` system. It's not a common thing at all, but in that case, just `arch` might be misleading.

Reply:
updated it